src/Cli/ConfigGenerator.cs

```diff
@@ -32,24 +32,35 @@ public class ConfigGenerator
         /// </summary>
         public static bool TryGenerateConfig(InitOptions options)
         {
-            if (!TryGetConfigFileBasedOnCliPrecedence(options.Config, out string runtimeConfigFile))
+            string runtimeConfigFile = RuntimeConfigPath.DefaultName;
+            string? environmentValue = Environment.GetEnvironmentVariable(RuntimeConfigPath.RUNTIME_ENVIRONMENT_VAR_NAME);
```

Comment:
Can we GetEnvironmentVariable in the else?
This saves us 1 call if user already provides a config file

Reply:
we need the value before that since we are validating the condition in elseif

Reply:
We could write the else as 

```
else
{
envValue =  GetEnvironmentVariable()
 if (envValue)
}
```

Reply:
but we have another else as well, which should only execute when the above two conditions are not met.